lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-amazon-cloud_manager-virtual_template.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Amazon_CloudManager_VirtualTemplate <
+      MiqAeServiceManageIQ_Providers_CloudManager_Template
```

Comment:
Prefer keeping this on one line...I find this so strange to read as two separate lines.  Yes, rubocop probably complains about line length, but this is a case where we ignore the bot because of the readability.